src/doc/src/commands/cargo-run.md

```diff
@@ -65,7 +65,7 @@ for more details.
 
 <dl>
 
-<dt class="option-term" id="option-cargo-run--F"><a class="option-anchor" href="#option-cargo-run--F"></a><code>-F</code> <em>features</em></dt>
```

Comment:
This change doesn't seem to be correct. The flag is uppercase.

Reply:
@ehuss Maybe make sense to remove it, because this option is not mentioned at doc https://doc.rust-lang.org/stable/cargo/commands/cargo-run.html and I don't see it at cargo 1.60.0 (d1fd9fe2c 2022-03-01) (see just --features)

Reply:
It's a new flag, added in 1.61.  The code here has not yet reached stable. You can view the nightly docs by adding `nightly` to the url like: https://doc.rust-lang.org/nightly/cargo/commands/cargo-run.html

Reply:
@ehuss Ok, reverted. Curious, is there reason why all options are lowercase and just this is uppercase?

Reply:
There are other uppercase options like `-Z` and `-V`.  I think uppercase `-F` was chosen to avoid possible confusion with lowercase `-f` which is often used to mean `--force` or `--file` in many unix utilities.